pages/speaking/tutorial.vue

```diff
@@ -117,5 +117,6 @@ export default {
     @media (min-width: 1024px) {
         @apply px-6;
     }
+    @apply xl:px-12;
```

Comment:
I checked the design spec again, and found out that the padding in the larger screen is a little different from the page (/speaking/recording).

Reply:
Nice, it looks great!